container-images/foreman/Containerfile

```diff
@@ -17,3 +13,8 @@ ENV RAILS_SERVE_STATIC_FILES=true
 ENV RAILS_ENV=production
 CMD /usr/share/foreman/bin/rails db:migrate && /usr/share/foreman/bin/rails db:seed && /usr/share/foreman/bin/rails server --environment production
 EXPOSE 3000/tcp
+
```

Comment:
Note line 8 where it switches to the `foreman` user and thus breaks the ability to run dnf.